modules/oe_content_publication/config/install/field.storage.node.oe_publication_type.yml

```diff
@@ -13,7 +13,7 @@ settings:
 module: rdf_skos
 locked: false
 cardinality: -1
-translatable: true
+translatable: false
```

Comment:
As it is existing field we need to provide update path for this setting in post_update().

Reply:
I discussed the possible implementation with @drishu and we agreed that this change should be in accordance with the other SKOS configurations. I reverted the setting back to `true`.